agenda.php

```diff
@@ -309,6 +316,89 @@ protected function getctag() {
         return NULL;
     }
     
+    //if add is true, then add $usermail to the event, otherwise, remove it.
+    function updateAttendee($url, $usermail, $add, $attendee_CN=NULL) {
+        $raw = file_get_contents_safe('./data/' . $url);
+        $etag = file_get_contents_safe('./data/' . $url . '.etag');
+        
+        $vcal = \Sabre\VObject\Reader::read($raw);
+        
+        if($add) {
+            $already_in = false;
+            if(isset($vcal->VEVENT->ATTENDEE)) {
+                foreach($vcal->VEVENT->ATTENDEE as $attendee) {
+                    if(strpos((string)$attendee, $usermail) >= 0) {
+                        $already_in = true;
```

Comment:
Idea while I'm reading this code: perhaps it would be simpler to replace those lines 331 & 332 with

```
  $this->log->info("Try to add a already registered attendee");
  return;
```
that could simplify the code a bit (because we could get rid of the variable  `$already_in`  and we could get rid of the `if` below).
But since you have a longer term vision of this code than me, feel completely free to disregard this idea if it does not make sense

Reply:
Indeed, fixed in b814854